src/main/resources/schema.sql

```diff
@@ -9,11 +10,21 @@ CREATE TABLE PREFECTURE (
     PRIMARY KEY (NUMBER)
 );
 
+CREATE TABLE GARBAGE_TYPE (
+    TYPE_ID INT NOT NULL,
+    TYPE_NAME VARCHAR(50) NOT NULL,
```

Comment:
[nits]これがもし重複しないなら unique を付けることで整合性が保たれるので、よかったら検討を。
```suggestion
    TYPE_NAME VARCHAR(50) NOT NULL UNIQUE,
```

Reply:
TYPE_NAMEは重複しないためUNIQUE制約を付与しました